specification/logic/resource-manager/Microsoft.Logic/stable/2016-06-01/examples/WorkflowsCreateOrUpdate.json

```diff
@@ -0,0 +1,125 @@
+{
+  "parameters": {
+    "api-version": "2016-06-01",
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "test-resource-group",
+    "workflowName": "test-workflow",
+    "workflow": {
+      "properties": {
+        "integrationAccount": {
+          "name": "test-integration-account",
+          "id": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/test-resource-group/providers/Microsoft.Logic/integrationAccounts/test-integration-account",
+          "type": "Microsoft.Logic/integrationAccounts"
+        },
+        "definition": {
+          "$schema": "https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#",
+          "contentVersion": "1.0.0.0",
+          "parameters": {
+            "$connections": {
+              "defaultValue": {},
+              "type": "Object"
+            }
+          },
+          "triggers": {
+            "manual": {
+              "type": "Request",
+              "kind": "Http",
+              "inputs": {
+                "schema": {}
+              }
+            }
+          },
+          "actions": {
+            "Find_pet_by_ID": {
+              "runAfter": {},
+              "type": "ApiConnection",
+              "inputs": {
+                "host": {
+                  "connection": {
+                    "name": "@parameters('$connections')['test-custom-connector']['connectionId']"
+                  }
+                },
+                "method": "get",
+                "path": "/pet/@{encodeURIComponent('1')}"
+              }
+            }
+          },
+          "outputs": {}
+        },
+        "parameters": {
+          "$connections": {
+            "value": {
+              "test-custom-connector": {
+                "connectionId": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/test-resource-group/providers/Microsoft.Web/connections/test-custom-connector",
+                "connectionName": "test-custom-connector",
+                "id": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/providers/Microsoft.Web/locations/brazilsouth/managedApis/test-custom-connector"
+              }
+            }
+          }
+        }
+      },
+      "location": "brazilsouth",
+      "tags": {}
+    }
+  },
+  "responses": {
+    "200": {
```

Comment:
Please also add an example response for 201.

Reply:
Added